local-proxy/src/server.ts

```diff
@@ -1,4 +1,4 @@
-import { join as pathJoin } from 'path';
+import { join as pathJoin, resolve as pathResolve } from 'path';
```

Comment:
`import * as path from 'path'` would let you use `path.join` and `path.resolve`, with *_less typing_*!!1!

Reply:
There's a variable called path in the code below, it would shadow the path module.